cmake/AzAutoGen.py

```diff
@@ -248,11 +248,18 @@ def ProcessTemplateConversion(autogenConfig, dataInputSet, dataInputFiles, templ
                     currentFile.truncate()
                     with open(outputFile, 'w+') as currentFile:
                         currentFile.write(compareFD.getvalue())
-                        print('Generating %s with template %s and inputs %s' % (outputFile, templateFile, ", ".join(dataInputFiles)))
+                        if autogenConfig.verbose == True:
+                            print('Generating %s with template %s and inputs %s' % (outputFile, templateFile, ", ".join(dataInputFiles)))
```

Comment:
Minor: I think the file is generated at this point.

Reply:
Can you update this to use f-strings
```suggestion
                            print(f'Generating {outputFile} with template {templateFile} and inputs {", ".join(dataInputFiles)}')
```